drivers/mrf24j40/include/mrf24j40_internal.h

```diff
@@ -33,7 +33,7 @@ extern "C" {
 /**
  * @brief initialization as decribed in datasheet
  */
-void mrf24j40_init(mrf24j40_t *dev);
+int mrf24j40_init(mrf24j40_t *dev);
```

Comment:
The return values are missing in the docs.

Reply:
Added it to the docs.